### PR TITLE
Test were added for the search and order query

### PR DESCRIPTION
**Pull request by mpushnikA1QA**

### CODE REVIEW

File: src/test/steps/sortAndOrderParameters.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  $randomSearchValue = getMultipleRandom(randomSearchData[$randomSearchKey], 1)[0];
+  $randomSortParameter = getMultipleRandom(SORT_FIELDS[this.entity], 1)[0];
+  let requestUrl = `${SEARCH_ENTITY_ENDPOINT(ENTITIES[entity])}${SEARCH_PARAMETER_ENDPOINT($randomSearchKey, $randomSearchValue)}${SORT_PARAMETER_ENDPOINT($randomSortParameter)}${ORDER_PARAMETER_ENDPOINT(this.order)}`;
+  const response = await $sortAndOrderAPIClient.getRequest(requestUrl);
+  $statusCode = response.status;
+  $responseBodyItems = response.data.items;
+  $totalRecordCount = response.data.meta.totalRecordCount;
+});
+
+Then(/^Check that the data is sorted correctly and that it is displayed in the correct order - '(.*)'$/, async (order) => {
```
Comment (danil-shamsiev) [inline src/test/steps/sortAndOrderParameters.js:24]: `async` is used, but no `await` is present in code block

File: src/test/steps/sortAndOrderParameters.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      }
+    }
+  } else {
+    expectChai.fail(`This order [${order}] is not supported.`);
+  }
+
+  expectChai(tempArray.length).to.equal(0, `For the [${this.entity}] entity for ids - [${tempArray.toString()}], should be sort by parameter [${this.randomSortParameter}]`);
+});
+
+Then(/^Check that the data is sorted correctly without specified order$/, async () => {
```
Comment (danil-shamsiev) [inline src/test/steps/sortAndOrderParameters.js:67]: `async` is used, but no `await` is present in code block

File: src/test/steps/sortAndOrderParameters.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  }
+  expectChai(tempArray.length).to.equal(0, `For the [${this.entity}] entity for ids - [${tempArray.toString()}], should be sort by parameter [${this.randomSortParameter}]`);
+});
+
+Then(/^Check that the number of elements in the response is greater than zero for '(.*)'$/, async (entity) => {
+  const itemsLength = $responseBodyItems.length;
+  expectChai(itemsLength).to.be.gt(0, `Total number of elements should be greater than zero for [${entity}] entity`);
+});
+
+Then(/^Check that the total record count of elements in the response result is greater than zero for '(.*)'$/, async (entity) => {
```
Comment (danil-shamsiev) [inline src/test/steps/sortAndOrderParameters.js:94]: `async` is used, but no `await` is present in code block

**Review by danil-shamsiev — CHANGES_REQUESTED** [review]

File: src/test/steps/sortAndOrderParameters.js (hunk 3 above)
Comment (mpushnikA1QA) [reply src/test/steps/sortAndOrderParameters.js]: Fixed

File: src/test/steps/sortAndOrderParameters.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+      const secondDate = $responseBodyItems[i + 1][this.randomSortParameter];
+      if (new Date(firstDate).getTime() > new Date(secondDate).getTime()) {
+        tempArray.push($responseBodyItems[i][entityId]);
+      }
+    }
+  }
+  expectChai(tempArray.length).to.equal(0, `For the [${this.entity}] entity for ids - [${tempArray.toString()}], should be sort by parameter [${this.randomSortParameter}]`);
+});
+
+Then(/^Check that the number of elements in the response is greater than zero for '(.*)'$/, async (entity) => {
```
[the remark this answers is not in the document]
Comment (mpushnikA1QA) [reply src/test/steps/sortAndOrderParameters.js]: Done

File: src/test/steps/sortAndOrderParameters.js (hunk 1 above)
Comment (mpushnikA1QA) [reply src/test/steps/sortAndOrderParameters.js]: Fixed

**Review by danil-shamsiev — APPROVED** [review]

**Pull request merged**